packages/sources/src/styles/subform.scss

```diff
@@ -0,0 +1,3 @@
+.ins-c-sources__wizard--subform-nested {
+    padding-left: var(--pf-global--spacer--md);
```

Comment:
you can use pf class instead `pf-u-pl-md` and use it instead. This class does not need to exist.

Reply:
Thanks! :taco: